packages/server/api.js

```diff
@@ -487,7 +490,7 @@ module.exports = async function attachAPI(app, {wss, db, dbDir}) {
             // the username variable is passed. To make this a little less
             // evil, it's possible for that word to be passed manually
             // (as the second argument to requireNameValid).
```

Comment:
Does this comment still apply?

Reply:
Nope, thanks. Removed (commit to be pushed).

Funnily enough, I don't think anything actually passed an `errorFieldName` to `requireNameValid`.